installer/install.iss

```diff
@@ -2788,16 +2810,28 @@ begin
         GitSystemConfigSet('pull.ff','only')
     end;
 
+    {
+        Configure credential helper
+    }
+
+    if RdbGitCredentialManager[GCM_None].checked then begin
+        GitSystemConfigSet('credential.helper',#0);
+        GitSystemConfigSet('credential.https://dev.azure.com.useHttpPath',#0);
+    end else if RdbGitCredentialManager[GCM_Classic].checked then begin
+        GitSystemConfigSet('credential.helper','manager');
+        GitSystemConfigSet('credential.https://dev.azure.com.useHttpPath',#0);
```

Comment:
Sneaky ;-)

Reply:
Good sneaky or bad sneaky? This setting is required for GCM Core since it doesn't include the "walk the process tree" workaround that GCM for Windows does.

Reply:
Ah... That's a good explanation ;-)